Makefile

```diff
@@ -19,6 +18,12 @@ export GOPATH   := $(shell go env GOPATH)
 WORKING_DIR     := $(shell pwd)
 TESTPARALLELISM := 4
 
+# Override during CI using `make [TARGET] PROVIDER_VERSION=""` or by setting a PROVIDER_VERSION environment variable
+# Local & branch builds will just used this fixed default version unless specified
+PROVIDER_VERSION ?= 1.0.0-alpha.0+dev
```

Comment:
Doesn't need to match on major? For Go sdkgen?

Reply:
v0 and v1 are compatible for Go SDK Gen - they'll both have the same root. We could have this set to 0.0.1-alpha.0+dev potentially but think either should work fine.